packages/synapse-react-client/package.json

```diff
@@ -78,6 +76,7 @@
     "mui-one-time-password-input": "1.1.0",
     "plotly.js-basic-dist": "^2.24.3",
     "pluralize": "^8.0.0",
+    "prop-types": "^15.8.1",
```

Comment:
For some reason, `tsup` isn't bundling/properly writing a dynamic import for the `prop-types` dependency, which is imported via `react-switch` (but also many other dependencies?)

This issue resolves if we include it in our dependencies.